tests/quantizers_infrastructure_tests/pytorch_tests/base_pytorch_infrastructure_test.py

```diff
@@ -0,0 +1,128 @@
+
+        Args:
+            quantization_config: node quantization config class
+        """
+        super().__init__(quantization_config,
+                         qi.QuantizationTarget.Weights,
+                         [qi.QuantizationMethod.POWER_OF_TWO, qi.QuantizationMethod.SYMMETRIC])
+
+    def initialize_quantization(self,
+                                tensor_shape: torch.Size,
+                                name: str,
+                                layer: nn.Module) -> Dict[str, nn.Parameter]:
+        return
+
+    def __call__(self,
+                 inputs: nn.Parameter,
+                 training: bool) -> nn.Parameter:
+
+        return inputs * 0
+
+
+class ZeroActivationsQuantizer(qi.BasePytorchTrainableQuantizer):
+    """
+    Trainable constrained quantizer to quantize a layer inputs.
+    """
+
+    def __init__(self, quantization_config: NodeWeightsQuantizationConfig):
+        """
+        Initialize a TrainableWeightQuantizer object with parameters to use
+        for the quantization.
+
+        Args:
+            quantization_config: node quantization config class
+        """
+        super().__init__(quantization_config,
+                         qi.QuantizationTarget.Activation,
+                         [qi.QuantizationMethod.POWER_OF_TWO, qi.QuantizationMethod.SYMMETRIC])
+
+    def initialize_quantization(self,
+                                tensor_shape: torch.Size,
+                                name: str,
+                                layer: nn.Module) -> Dict[str, nn.Parameter]:
+        return
+
+    def __call__(self,
+                 inputs: nn.Parameter,
+                 training: bool = True) -> nn.Parameter:
+
+        return inputs * 0
+
+
+def dummy_fn():
+    return
+
+
+class BasePytorchInfrastructureTest:
+
+    def __init__(self,
+                 unit_test,
+                 num_calibration_iter=1,
+                 val_batch_size=1,
+                 num_of_inputs=1,
+                 input_shape=(3, 8, 8)):
+        self.unit_test = unit_test
+        self.val_batch_size = val_batch_size
+        self.num_calibration_iter = num_calibration_iter
+        self.num_of_inputs = num_of_inputs
+        self.input_shape = (val_batch_size,) + input_shape
+
+    def generate_inputs(self):
+        return [np.random.randn(*in_shape) for in_shape in self.get_input_shapes()]
+
+    def get_input_shapes(self):
+        return [self.input_shape for _ in range(self.num_of_inputs)]
+
+    def get_dispatcher(self, weight_quantizers=None, activation_quantizers=None):
+        return qi.PytorchNodeQuantizationDispatcher(weight_quantizers, activation_quantizers)
+
+    def get_wrapper(self, layer, dispatcher):
+        return qi.PytorchQuantizationWrapper(layer, dispatcher)
+
+    def get_weights_quantization_config(self):
+        op_cfg, _ = get_op_quantization_configs()
+        qc = QuantizationConfig()
+        return NodeWeightsQuantizationConfig(qc, op_cfg, dummy_fn, dummy_fn, -1)
```

Comment:
Consider adding explicit arguments names in this call (for instance what it the "-1" for?)